README.md

```diff
@@ -446,3 +447,31 @@ Type: `number`. Default: `8080`.
 
 Preferred port for development servers.
 If the specified port is unavailable, another port is used.
+
+### browserslist
+
+Type: `Array<string>` \| `Object`. A valid [Browserslist](https://github.com/browserslist/browserslist) value. Default:`['> 0.5%', 'last 2 versions', 'Firefox ESR', 'not dead']`.
+
+This value is used by Autoprefixer to set vendor prefixes in the CSS of your stylesheets, and is used to determine Babel compilation via [babel-preset-env](#babel).
+
+The default value uses Browserslist's default, which is `> 0.5%, last 2 versions, Firefox ESR, not dead`.
```

Comment:
Since we already state the default above, I think we could end this sentence at "Browserslist's default."